lib/foreman/renderer/scope/variables/base.rb

```diff
@@ -75,7 +75,7 @@ def kickstart_attributes
             @arch      = architecture_name
             @osver     = major.try(:to_i)
             @mediapath = mediumpath(@medium_provider) if @medium_provider
-            @repos     = repos(host)
+            @repos     = @medium_provider&.additional_media
```

Comment:
Is it possible that `@additional_media` are loaded for us already by `load_variables_base` on l.33?

Reply:
good point. Actually, i think this variable should be removed along with the method, it doesn't look like it's being used in the default templates.